packages/dds/map/src/test/memory/map.spec.ts

```diff
@@ -57,30 +33,49 @@ describe("SharedMap memory usage", () => {
         // See the comment at the top of the test suite for more details.
     });
 
-    benchmarkMemory({
-        title: "Create empty map",
-        minSampleCount: 1000,
-        benchmarkFn: async () => {
-            createLocalMap("testMap");
-        },
-    });
+    benchmarkMemory(new class implements IMemoryTestObject {
```

Comment:
Add `minSampleCount = 500` here too, and in the corresponding test in Matrix and SharedString.